src/widgets/benefits/ui/benefits.module.scss

```diff
@@ -0,0 +1,114 @@
+.benefits-grid {
+  display: grid;
+  grid-template-areas:
+    'top-left top-center top-right'
+    'bottom-left bottom-left bottom-right';
+  grid-template-columns: repeat(3, 1fr);
+  gap: 30px;
+
+  list-style-type: none;
+
+  .grid-item {
+    padding: 24px;
+
+    font-size: 18px;
+    line-height: 28px;
+    color: $color-gray-600;
+
+    background-color: $color-yellow-50;
+    border: 1px solid rgb(255 219 32 / #{$opacity-8});
```

Comment:
Should it be extracted into a separate variable?
And we use HSL instead of RGB

Reply:
Corrected